supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -46,10 +48,107 @@
 if TYPE_CHECKING:
     from typing import List, Union
     from supertokens_python import AppInfo
-    from supertokens_python.querier import Querier
 
-from .constants import JWKCacheMaxAgeInMs, JWKRequestCooldownInMs
 from .interfaces import SessionContainer
+from supertokens_python.querier import Querier
+
+
+class GetJWKSResult:
+    def __init__(self, keys: List[PyJWK], last_fetched: int):
+        self.keys = keys
+        self.last_fetched = last_fetched
+
+
+from typing_extensions import TypedDict
+import threading
+from os import environ
+
+
+class JWKSConfigType(TypedDict):
+    cache_max_age: int
+    refresh_rate_limit: int
+
+
+JWKSConfig: JWKSConfigType = {
+    "cache_max_age": 6000,
+    "refresh_rate_limit": 500,
+}
+
+jwks_cache: Optional[GetJWKSResult] = None
+mutex = threading.RLock()
+
+
+def get_jwks_cache():
+    return jwks_cache
+
+
+def reset_jwks_cache():
+    global jwks_cache
+    jwks_cache = None
+
+
+urls_attempted_for_jwks_fetch: List[str] = []
```

Comment:
We should comment above variables we use for testing so they dont accidentally get used in the sdk in the future

Reply:
Also is this used anywhere?